### PR TITLE
#88269

### PR DESCRIPTION
**Pull request by patrykbura**

### CODE REVIEW

File: Magento_GiftCardAccount/web/template/summary/gift-card-account.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                </a>
+            </div>
+        </div>
+        <span
+            class="order-summary__value"
+            data-bind="text: '-' + $parent.getAmount(a)"
+        >
+        </span>
+    </li>
+<!-- /ko -->
```
Comment (JKrupinski) [inline Magento_GiftCardAccount/web/template/summary/gift-card-account.html:27]: fix indentation

File: Magento_GiftCardAccount/web/template/summary/gift-card-account.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!-- ko if: isAvailable() -->
+    <!-- ko foreach: getAppliedGiftCards() -->
+    <li class="order-summary__list-item">
+        <div>
+            <span class="order-summary__label">
+                <!-- ko text: $parent.title + ' (' + c + ')' --><!-- /ko -->
+            </span>
+            <div>
+                <a
```
Comment (JKrupinski) [inline Magento_GiftCardAccount/web/template/summary/gift-card-account.html:9]: Could you check if you can easly change it to `button`?

Comment (JKrupinski) [reply Magento_GiftCardAccount/web/template/summary/gift-card-account.html:9]: `button--link` class

File: Magento_GiftCardAccount/web/template/summary/gift-card-account.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!-- ko if: isAvailable() -->
+    <!-- ko foreach: getAppliedGiftCards() -->
+    <li class="order-summary__list-item">
```
Comment (JKrupinski) [inline Magento_GiftCardAccount/web/template/summary/gift-card-account.html:3]: fix indentation

**Review by JKrupinski — CHANGES_REQUESTED** [review]

File: Magento_GiftCardAccount/web/template/summary/gift-card-account.html (hunk 2 above)
Comment (patrykbura) [reply Magento_GiftCardAccount/web/template/summary/gift-card-account.html]: I'll add `button--link` class. But it is `<a>` tag https://magento2-alpaca-theme-git-master-snowdog1.vercel.app/components/detail/button--link.html

File: Magento_GiftCardAccount/web/template/summary/gift-card-account.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        <li class="order-summary__list-item">
+            <div>
+                <span class="order-summary__label">
+                    <!-- ko text: $parent.title + ' (' + c + ')' --><!-- /ko -->
+                </span>
+                <div>
+                    <button
+                        class="button--link-secondary"
+                        type="button"
+                        aria-label="Remove gift card"
```
Comment (JKrupinski) [inline Magento_GiftCardAccount/web/template/summary/gift-card-account.html:12]: please use translation $t(), move it to data-bind

**Review by JKrupinski — APPROVED** [review]

**Pull request merged**